android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -136,6 +136,20 @@ private void WriteJsonElement(ByteArrayOutputStream stream, JsonElement value){
         }
     }
 
+    /**
+     * Converts Map to JsonObject, ArrayList to JsonArray and
+     * returns null if these 2 types are a no-match
+     * */
+    static JsonElement readValueAsJsonElement(final Object object) {
+        Gson gson = new Gson();
```

Comment:
I recognise the scope is short here but I would still prefer this reference were `final`.

Reply:
Sure, thanks for pointing this out. I'm making variables final whichever I am coming across and feel is appropriate 😄 

Addressed 59ff55b9cce0473920b322e21f6b9297a3954574